src/main.cpp

```diff
@@ -3041,13 +3042,13 @@ void UnlinkPrunedFiles(std::set<int>& setFilesToPrune)
 }
 
 /* Calculate the block/rev files that should be deleted to remain under target*/
-void FindFilesToPrune(std::set<int>& setFilesToPrune)
+void FindFilesToPrune(std::set<int>& setFilesToPrune, uint64_t nPruneAfterHeight)
```

Comment:
I'm not sure it was a good idea to put nPruneAfterHeight in CChainParams in the first place. In any case, for functions that only take one thing from CChainParams (not Consensus::Params) I prefer to use the single thing directly instead of the full CChainParams, and if FlushStateToDisk ever needs more things from CChainParams, he can change the function prototype himself. See https://github.com/bitcoin/bitcoin/pull/6173 for example.
So I'm going to leave it as is and squash.